test/evolutionaryAlgorithm/test_EvolutionSimulatorBuilder.py

```diff
@@ -27,7 +27,7 @@ def test_create_evolution_simulator_from_dict():
                         "Sigma": 1.0,
                         "Probability" : 0.5
                     }},
-        "ParentSelector": {"Type": "simpleParentSelector"},
+        "ParentSelector": {"Type": "rouletteParentSelector"},
```

Comment:
Jeśli chcemy kożystać z obu selektorów należy napisać drugi test(żeby przetestować oba)

Reply:
Nie powinniśmy raczej z obu ja bym zostawił tuletkowe

Reply:
Okej, to kod musi się kompilować z rulete parent selector a obecnie rulete parent factory zwraca not implemented error